src/App.tsx

```diff
@@ -1,10 +1,67 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { request } from './utils/fetchClient';
+import { Todo, TodoStatus } from './types/Todo';
+import { ErrorNotification } from './ErrorNotification';
+import { TodosMain } from './TodosMain';
+import { TodosHeader } from './TodosHeader';
+import { TodosFooter } from './TodosFooter';
 
-const USER_ID = 0;
+const USER_ID = 10332;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [statusFileter, setStatusFilter] = useState<
+  TodoStatus.All
+  | TodoStatus.Completed
+  | TodoStatus.Uncompleted
+  >(TodoStatus.All);
+
+  const loadTodos = async () => {
+    try {
+      const response = await request<Todo[]>('/todos?userId=10332');
+
+      setTodos([...todos, ...response]);
```

Comment:
Consider using 'setTodos(response)' instead of 'setTodos([...todos, ...response])' since the 'todos' state should be replaced with the new response, not merged.